crates/ruma-common/src/serde/base64.rs

```diff
@@ -57,8 +57,9 @@ impl Base64Config for UrlSafe {
 
 impl<C: Base64Config, B> Base64<C, B> {
     // See https://github.com/matrix-org/matrix-spec/issues/838
-    const CONFIG: GeneralPurposeConfig =
-        general_purpose::NO_PAD.with_decode_allow_trailing_bits(true);
+    const CONFIG: GeneralPurposeConfig = general_purpose::NO_PAD
+        .with_decode_allow_trailing_bits(true)
```

Comment:
I think that comment should move above the `with_decode_allow_trailing_bits` line.